custom/icds/location_reassignment/parser.py

```diff
@@ -159,7 +159,7 @@ class HouseholdReassignmentParser(object):
     def __init__(self, domain, workbook):
         self.domain = domain
         self.workbook = workbook
-        self.reassignments = {}  # household id mapped to new awc code
+        self.reassignments = {}  # household id mapped to old and new site code
```

Comment:
It'd be helpful for this to state how the site codes are represented, like `household id mapped to a tuple of (old site code, new site code)` (I forget if that's accurate).

Reply:
https://github.com/dimagi/commcare-hq/pull/27134/commits/8cb6a9af26d958e1d1240913aa62a10ebd4867b7